roles/ceph-defaults/tasks/facts.yml

```diff
@@ -1,4 +1,13 @@
 ---
+- name: check if it is atomic host
+  stat:
+    path: /run/ostree-booted
+  register: stat_ostree
+
+- name: set_fact is_atomic
+  set_fact:
+    is_atomic: '{{ stat_ostree.stat.exists }}'
```

Comment:
We use double quotes everywhere else in the playbook, could you please do that instead of simple quotes?
Thanks

Reply:
I've got this updated.